spec/controllers/users_controller_spec.rb

```diff
@@ -365,7 +367,7 @@
         stub_cognito_response(method: :admin_get_user, payload: cognito_user)
         stub_cognito_response(method: :list_users_in_group, payload: cognito_users)
         expect_any_instance_of(AuthenticationBackend).to receive(:resend_invite)
```

Comment:
should we have something similar with the Notify call? I.e. testing it gets triggered

Reply:
Thanks, have added this.